spec/lib/insert/insert_import_spec.rb

```diff
@@ -251,7 +326,7 @@
   #     desigs = @donations.map{|d| d['designation']}
   #     expect(desigs.sort).to eq(Hamster::Vector["third party event", "third party event", "third party event", "third party event"])
   #   end
-
+  next
```

Comment:
Was that an accident?

Reply:
It was apparently! I'm going to remove all that commented stuff to clear it up too while we're here.

Reply:
I addressed this in: https://github.com/CommitChange/houdini/pull/104